doc/API.md

```diff
@@ -1,161 +1,8 @@
-
 # Torque API
 
+Torque.js is an efficient and stylish rendering method to animate your data. Torque.js uses TileCubes, which are JSON representations of multidimensional data with geospatial coordinates, to render data on the client.
 
-## L.TorqueLayer(options)
-
-One of two core classes for the Torque library - it is used to create an animated torque layer with custom settings.
-
-### Usage example
-
-```js
-  // initialize a torque layer that uses the CartoDB account details and SQL API to pull in data
-  var torqueLayer = new L.TorqueLayer({
-    user       : 'viz2',
-    table      : 'ow',
-    cartocss:  CARTOCSS
-  });
-```
-
-### Options
-
-##### Provider options
-
-| Option    | type       | Default   | Description                            | Options |
-|-----------|:-----------|:----------|:---------------------------------------|---------|
-| provider  | string     | ```sql_api```   | Where is the data coming from    | `sql_api`, `url_template`, or `windshaft` |
-
-##### CartoDB data options (SQL API provider)
-
-| Option    | type       | Default   | Description                            |
-|-----------|:-----------|:----------|:---------------------------------------|
-| user_name      | string     | ```null```      | CartoDB account name. Found from: http://accountname.cartodb.com|
-| table_name     | string     | ```null```      | CartoDB table name where data is found  |
-| query       | string     | ```null```      | SQL query to be performed to fetch the data. You must use this param or table, not at the same time |
-| cartocss  | string     | ```null```      | CartoCSS style for this map |
-| loop      | boolean    | ```true```      | If ```false```, the animation is paused when it reaches the last frame |
-
-
-### Time methods
-
-| Method    | options    | returns   | Description                            |
-|-----------|:-----------|:----------|:---------------------------------------|
-| ```setStep(step)``` | ```time numeric```    | ```this```   | sets the animation to the step indicated by ```step```, must be between 0 and ```steps```|
```

Comment:
Added link to http://docs.cartodb.com/cartodb-platform/cartocss/properties-for-torque/#torque-frame-count-number